corehq/apps/cloudcare/templates/formplayer/menu_list.html

```diff
@@ -128,15 +128,17 @@ <h1 class="page-title"><%- title %></h1>
 </script>
 
 <script type="text/template" id="form-menu-template">
-   <i class="fa fa-bars dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"></i>
-   <ul class="dropdown-menu dropdown-menu-right">
-     <% if (printOptionsEnabled) { %>
-     <li class="dropdown-header" id="print-header" tabindex="0" onclick="window.print()">{% trans "Print" %}</li>
-     <% } %>
-     <% if (languageOptionsEnabled) { %>
-     <li class="dropdown-header" tabindex="0">{% trans "Languages" %}</li>
-     <% } %>
-   </ul>
+  <li tabindex="0" id="form-menu-dropdown">
```

Comment:
why is this an `li` element and not say a `div`?

Reply:
fixed with [1043c20](https://github.com/dimagi/commcare-hq/pull/32641/commits/1043c207ce04e1a0e87e12d8479d3e707e25465a)